docs/FRSKY_R9_TX.md

```diff
@@ -0,0 +1,167 @@
+# mLRS Documentation: Frsky R9M Transmitter #
+
+([back to main page](../README.md))
+
+The Frsky R9M transmitter module is a a good choice for mLRS since it supports 1 W output power and is commercially available at an affordable price.
+
+<table>
+  <tbody>
+    <tr>
+      <td>Frequency Bands</td>
+      <td>868 MHz/915 MHz</td>
+    </tr>
+    <tr>
+      <td>Max. RF Output Power</td>
+      <td>30 dBm (1 W)</td>
+    </tr>
+    <tr>
+      <td>Supported Modes</td>
+      <td>19 Hz</td>
+    </tr>
+  </tbody>
+</table>
+
+Note that R9M module cannot connect with mLRS boards which support 868/915 MHz and use the SX126x LoRa chipset. This is because R9 hardware uses the SX127x LoRa chipset, which is incompatible with the SF6 spreading factor used by the mLRS 19 Hz mode. In addition, the SX127x does not support the SF5 spreading factor which is used for the mLRS 31 Hz mode.
```

Comment:
I wonder if we should go with "our" susual style and write "Note: The R9M...." ?

Reply:
Updated